lib/eigensolve_quda.cpp

```diff
@@ -337,8 +337,8 @@ namespace quda
     // C_1 is the current 'out' vector.
 
     // Clone 'in' to two temporary vectors.
-    auto tmp1 = std::make_unique<ColorSpinorField>(in);
-    auto tmp2 = std::make_unique<ColorSpinorField>(out);
+    auto tmp_1 = std::make_unique<ColorSpinorField>(in);
```

Comment:
Not sure what the point of the rename was here, and I don't care 😄 

But while you're there, you might as well use objects here and not unique pointers now we have a move constructor.

Reply:
I think at some point I was wrestling with a bug and thought that `tmp1` and `tmp2` aliasing other variables in the class was causing an issue, and never bothered to revert it after I hunted down the issue. I imagine I could change it back. In any case, I can switch to objects here! (assuming with the objects I can still use `std::swap`?)

Reply:
Swap works as long as objects are movable, so this all should just work.